src/rating-average/RatingAverage.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC } from 'react';
+
+import {
+  adjustDecimalPrecision,
+  formatToNumberWithCommas,
+  getTotalRatingCount,
+  getWeightedAverage
+} from '../utils';
+import { IRatingAverageProp } from './types';
+import starImg from '../assets/star-grey.svg';
+import PartiallyFilledStar from './PartiallyFilledStar';
+import { GenericElements, RATING_AVERAGE_DEFAULTS } from '../constants';
+import classes from './styles.module.scss';
+
+const { icon } = RATING_AVERAGE_DEFAULTS;
+
+const RatingAverage: FC<IRatingAverageProp> = (props) => {
+  const {
+    ratings,
+    customAverageFn,
```

Comment:
can't we have `customAverageFn = getWeightedAverage` and remove `computeAverage`?